### ISSUE TITLE
Error while encoding Json (_BaseJSONEncoder) 

### ISSUE THREAD

**Issue by ubbikk** [Yelp/py_zipkin#106]
I'm trying to use Encoding.V2_JSON \ Encoding.V1_JSON  but I'm receiving an exception:

```Traceback (most recent call last):
  File "/home/dmytro/anaconda3/lib/python3.6/site-packages/py_zipkin/logging_helper.py", line 129, in _emit_spans_with_span_sender
    report_timestamp=self.report_root_timestamp,
  File "/home/dmytro/anaconda3/lib/python3.6/site-packages/py_zipkin/logging_helper.py", line 180, in add_span
    encoded_span,
  File "/home/dmytro/anaconda3/lib/python3.6/site-packages/py_zipkin/encoding/_encoders.py", line 141, in fits
    return 2 + current_count + current_size + len(new_span) <= max_size
TypeError: unsupported operand type(s) for +: 'int' and 'list'
```

I use master branch, there is a similar exception for 0.14.1 (I switched to master hoping that the issue was fixed in master):

```Traceback (most recent call last):
  File "/home/dmytro/anaconda3/lib/python3.6/site-packages/py_zipkin/logging_helper.py", line 129, in _emit_spans_with_span_sender
    report_timestamp=self.report_root_timestamp,
  File "/home/dmytro/anaconda3/lib/python3.6/site-packages/py_zipkin/logging_helper.py", line 180, in add_span
    encoded_span,
  File "/home/dmytro/anaconda3/lib/python3.6/site-packages/py_zipkin/_encoding_helpers.py", line 343, in fits
    return 2 + current_count + current_size + len(new_span) <= max_size
TypeError: unsupported operand type(s) for +: 'int' and 'list'
```

I tried to debug a little: it seems `_BaseJSONEncoder.fits` expects an integer as the first argument but in `ZipkinBatchSender.add_span` we pass `self.queue` that is in fact a list

**Comment by drolando** [comment]
Thanks for the report! I'll try to fix this asap.

**Comment by drolando** [comment]
This should be fixed in `0.15.0`
